cdk/kittyhawk/test/ingress.test.ts

```diff
@@ -1,8 +1,48 @@
 import { Construct } from "constructs";
 import { domainToCertName, removeSubdomain } from "../src";
 import { Application } from "../src/application";
-import { failingTest } from "./utils";
+import { chartTest, failingTest } from "./utils";
 
+/*
+Ingress Port Behavior 
+
+For the application, the ingress port should never be explicitly defined. This is because the
+application port must be uniform with the ingress port provided. 
+
+1. Default Behavior
+If `port` IS NOT specified in `Application`, continue with default behavior in `Application` 
+and other children objects created. For `Ingress`, the default port is assumed.
+
+2. In Application, Not Ingress: Inherits Application port
+If `port` is specified in `Application`, the port value is used for the `Ingress` and `Application`.
+*/
+
+export function buildDefaultIngressChart(scope: Construct) {
+  new Application(scope, "serve", {
+    deployment: {
+      image: "pennlabs/website",
+      tag: "latest",
```

Comment:
I don't think this test actually creates an ingress

Reply:
Good catch :'D